backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/GitMacheteStatusTestSuite.java

```diff
@@ -41,7 +43,13 @@ protected void init(String scriptName) {
   }
 
   @Test
-  public void status() {
+  public void statusWithNoRemotes() {
```

Comment:
All these test cases are "the same" :confused: 

The parametrized test is an option:
https://github.com/junit-team/junit4/wiki/Parameterized-tests

This ofc can be postponed/delegated to another PR (I can do that somewhere btw :rofl: )

Reply:
Nice! Haven't found that before. Done on this PR